README.md

```diff
@@ -0,0 +1,149 @@
+
+```python
+        import pandas as pd
+        data = pd.read_csv('test.csv')
+        X = data.drop(columns=["target"]).values
+        y = data["target"].values
+```
+
+Once the data is ready, we can get a feature selector, fit it and look at the selected features:
+
+```python
+from omigami.omigami import FeatureSelector
+feature_selector = FeatureSelector(
+    repetitions=10,
+    n_outer=5,
+    n_inner=4
+    estimator="RFC",   # random forest classifier
+    metric="MISS",   # missclassifications
+)
+
+feature_selector.fit(X, y)
+
+selected_features = feature_selector.selected_features
+```
+
+It might take a while for it to complete, depending on your machine and on the model
+selected.
+
+The features are reported as column indexes. To get the names just pass the selection
+to the data frame:
+
+```python
+selected_feature_names = data.columns[list(selected_features["min"])]
+```
+
+### Parallelization
+
+The feature selection can be time consuming. To speed it up execute the various CV loops in parallel using a dask cluster.
+The dask cluster can be remote, or running in local to exploit all the processors of
+the your computer.
+
+For the latter case - which is probably the most common case - it's sufficient to run the following
+**at the beginning of your script**:
+
+```python
+from dask.distributed import Client
+client = Client()
+```
+
+*Also*: Dask gives you a neat dashboard to see the status of all the jobs at `http://localhost:8787/status`.
+
+## How it works
+
+![Schematic of repeated double cross-validation](https://global-uploads.webflow.com/5d3ec351b1eba4332d213004/5fd0e88651b733b656c3603b_ccukNlmNckEJ3p-Z9fHm2jPdgI9ILBDbOcdOBaBz3_WXA7VltferIk3vU1PPHztX5Gjcr0DMbh2xtvEK1lSYdou2xGAtni-Mq50W_cEpXssg2akHefa-H41jKDApZxctJlnVvk-b.png)
+
+1. The dataset is split into `n_outer` cross-validation splits.
+2. Each train split is further split into `n_inner` cross-validation splits.
+3. On each cross-validation split multivarate models are trained and evaluated.
+4. The least important fraction of features (`features_dropout_rate`) is removed, until there are no more features in the model
+5. The whole process is repeated `n_repetitions` times to improve the robustness of the selection.
+6. Parameters and features are averaged over all `n_outer` splits and all `repetitions`.
+
+## Parameters
+
+### `FeatureSelector` parameters
+
+- **repetitions**: Number of repetitions of the entire double cross-validation loop (default: `8`)
+- **n_outer**: Number of cross-validation splits in the outer loop
+- **n_inner**: Number of cross-validation splits in the inner loop (default: n_outer-1)
+- **estimator**: Multivariate model that you want to use for the feature selection. Supports
+  - `"RFC"`: Random Forest Classifier
+  - `"XGBC"`: XGBoost Classifier
+- **metric**: Metric to be used to assess fitness of estimators. Supports
+  - `"MISS"`: Number of missclassifications.
```

Comment:
not only, also all scikitlearn metrics

Reply:
how does a user pass them - simply by the string name, or differently?

Reply:
so far by string names. By the way ignore this comment, we'll have to fix something first, we'll update this README section once it's done

Reply:
we should add specific examples.